imports/plugins/core/ui-tagnav/client/components/tagNav.js

```diff
@@ -18,6 +18,21 @@ class TagNav extends Component {
     this.setState({ selectedTag: nextProps.selectedTag });
   }
 
+  /**
+   * onShopSelectChange
+   * @method
+   * @summary Handle change in selected shop
+   * @param {script} event
+   * @param {String} shopId - selected shopId
+   * @since 1.5.8
+   * @return {void}
+  */
+  onShopSelectChange = (event, shopId) => {
+    if (typeof this.props.handleShopSelectChange === "function") {
```

Comment:
don't we get this type of check from using PropTypes? i.e `PropTypes.func`. Is there a reason to specifically check here?